frontend/src/boot/i18n.ts

```diff
@@ -0,0 +1,24 @@
+import { boot } from 'quasar/wrappers';
+import messages from 'src/i18n';
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+import Quasar from 'quasar'
+
+Vue.use(VueI18n);
+console.log(Quasar.lang.getLocale())
+
+export const i18n = new VueI18n({
+  locale: Quasar.lang.getLocale(),
```

Comment:
Should we cache this to a local var instead of calling the method twice (once here and once in console.log)? I doubt there's a significant performance difference, but can the result possible change between the two calls?